ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java

```diff
@@ -271,7 +266,7 @@ public EventSeries getAuditTrail(EventQueryDefinition query) {
         EventSeries eventSeries = new EventSeries();
         try {
             jedis = getJedis();
-            String hashKey = getHashKey(query.getFrom(), EventConstants.ACTION_CHECK_OK);
+            String hashKey = getHashKey(query.getFrom());
             Set<String> events = jedis.zrangeByScore(hashKey, query.getFrom(), query.getTo(), 0, 100);
```

Comment:
4th and 5th parameters should be set in a constant and not hard coded

Reply:
That's right, but that was not the goal of this PR